app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+
+  def create 
+    @job = Job.find(params[:job_id])
+    @comment = Comment.new(comment_params)
+    @comment.job_id = @job.id
```

Comment:
This is not the most preferrable way to create objects that are associated to each other. Because a job has many comments, we can do...

```ruby
@comment = job.comments.new(comment_params)
```

this way leverages our associations 👍